image/minikube.json

```diff
@@ -0,0 +1,41 @@
+{
+    "variables": {
+        "aws_access_key": "{{env `AWS_ACCESS_KEY_ID`}}",
+        "aws_secret_key": "{{env `AWS_SECRET_ACCESS_KEY`}}"
+    },
+    "builders": [{
+        "type": "amazon-ebs",
+        "access_key": "{{user `aws_access_key`}}",
+        "secret_key": "{{user `aws_secret_key`}}",
+        "region": "eu-central-1",
+        "source_ami": "ami-1e339e71",
+        "instance_type": "t2.small",
+        "ssh_username": "ubuntu",
+        "ami_name": "shipyard minikube {{timestamp}}"
+    }],
+
+    "provisioners": [{
+        "type": "shell",
+        "inline": [
+            "curl -s -LO https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl",
+            "chmod +x ./kubectl",
+            "sudo mv ./kubectl /usr/local/bin",
+
+            "curl -s -Lo minikube https://storage.googleapis.com/minikube/releases/v0.22.2/minikube-linux-amd64",
+            "chmod +x ./minikube",
+            "sudo mv ./minikube /usr/local/bin",
+
+            "sudo apt update",
+            "sudo apt install -y apt-transport-https ca-certificates software-properties-common",
+            "curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -",
+            "sudo add-apt-repository \"deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable\"",
+            "sudo apt update",
+            "sudo apt install -y docker-ce",
+
+            "echo FIX ME: two calls to 'minikube start' are needed, the first one doesn't create the required systemd units",
```

Comment:
Why is this? Is there an issue for this in minikube and for us to fix it once it is fixed upstream?

Reply:
No idea why this happens and it doesn't seem to be reported, will create an issue as soon as i grab the logs, then reference it from the code.